performance_manager/lib/postgres_schema.py

```diff
@@ -25,14 +25,15 @@ class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
     hash = sa.Column(
         sa.LargeBinary(16), nullable=False, index=True, unique=False
     )
+    updated_on = sa.Column(sa.TIMESTAMP, server_default=sa.func.now())
```

Comment:
add `update_on` column to schema, default value handles setting value on INSERT, trigger sets value on UPDATE